src/Ilios/CoreBundle/Entity/User.php

```diff
@@ -1197,4 +1197,25 @@ public function getEncoderName()
 
         return null; // use the default encoder
     }
+    
+    /**
+     * Get all the schools an user is affiliated with
+     * This is used in granting view permissions for faculty
+     * in schools that are a users secondary cohort
+     * 
+     * @return ArrayCollection[School]
+     */
+    public function getAllSchools()
```

Comment:
please pull the method declaration up into `UserInterface` for interface/implementation parity.